Dockerfile

```diff
@@ -62,13 +65,14 @@ RUN export DEBIAN_FRONTEND='noninteractive' && \
     sdkmanager \
       "platforms;android-22" \
       "platforms;android-${ANDROID_SDK_PLATFORM_VERSION}" \
```

Comment:
I think once we go back to only installing 1 platform it may just fit in the cache

Reply:
Cache works now. Managed to uninstall  android emulators which get installed by default.